docs/reference/slate/change.md

```diff
@@ -555,6 +555,12 @@ Wrap the given node in a [`Block`](./block.md) node that match `properties`. For
 
 Wrap the given node in a [`Inline`](./inline.md) node that match `properties`. For convenience, you can pass a `type` string or `properties` object.
 
+### `wrapNodeByKey`
+
+`wraNodeByKey(key: String, parent: Node) => Change` <br/>
+
+Wrap the node with the specified key the parent [`Node`](./node.md). This will clear all children of the parent.
```

Comment:
This reads a bit weird:

Reply:
I don't know if there's actually a better way to describe this, though

Reply:
Yeah that makes sense!